Themes/default/languages/Profile.english.php

```diff
@@ -533,6 +533,12 @@
 $txt['trackEdit_action_id_group'] = 'Primary Membergroup';
 $txt['trackEdit_action_additional_groups'] = 'Additional Membergroups';
 
+// Registration Agreement
+$txt['trackEdit_action_agreement_accepted'] = 'Accepted the registration agreement';
+$txt['trackEdit_action_policy_accepted'] = 'Accepted the privacy policy';
+
+$txt['export_profile_data'] = 'Download profile data';
```

Comment:
This can be removed. UI strings for the data export function are covered elsewhere.